projects/workflows-creator/src/lib/builder/builder.component.ts

```diff
@@ -149,21 +142,23 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
     this.nodes
       .getGroups(true, NodeTypes.ACTION, true)
       .forEach(group => this.elseActionGroups.push(group));
-
-    this.localizationSvc.setLocalizedStrings(this.localizedStringMap);
     this.cdr.detectChanges();
   }
-
   /**
    * > If the diagram and state have changed, restore the diagram and state from the builder
    * @param {SimpleChanges} changes - SimpleChanges - the changes that have occurred in the component
    */
   async ngOnChanges(changes: SimpleChanges) {
+    if (changes['localizedStringMap'] && this.localizedStringMap) {
+      this.localizationSvc.setLocalizedStrings(this.localizedStringMap);
```

Comment:
why do you need this line if is already the first line of initiatenode

Reply:
If user changes any localized string after Init Thats y we code here in ngOnChanges

Reply:
removed.